aiven/client/cli.py

```diff
@@ -5587,6 +5590,83 @@ def organization__user__invite(self) -> None:
         """Invite user to join an organization"""
         self.client.invite_organization_user(self.args.organization_id, self.args.email)
 
+    @arg.json
+    @arg.organization_id
```

Comment:
Not specifying an organization id crashes things as we are passing `None` down the stack. We should make `organization-id` option mandatory for these commands, I don't think we have anything else using this so we could just add `required=True` (or create a parallel one called `organization_id_mandatory`).